src/brsTypes/components/LayoutGroup.ts

```diff
@@ -0,0 +1,20 @@
+import { FieldModel } from "./RoSGNode";
+import { AAMember } from "./RoAssociativeArray";
+import { Group } from "./Group";
+
+export class LayoutGroup extends Group {
```

Comment:
We'll eventually need to implement `boundingRect()` on this differently than we do the other components, simply because of the fact that `itemSpacings` has to be included.  But since we haven't implemented that _anywhere_ yet, it doesn't matter just yet! 🎉